akka-http-core/src/main/scala/akka/http/impl/engine/http2/Http2ServerDemux.scala

```diff
@@ -138,6 +138,14 @@ private[http2] class Http2ServerDemux(http2Settings: Http2CommonSettings, initia
       }
       def pullFrameIn(): Unit = if (allowReadingIncomingFrames && !hasBeenPulled(frameIn)) pull(frameIn)
 
+      def pullOutgoingSubStreams: Unit = {
```

Comment:
Maybe

```suggestion
      def pullOutgoingSubStreamsIfPossible(): Unit = {
```

or 

```suggestion
      def tryPullSubStreams(): Unit = {
```

to signify that we might not pull.